### ISSUE TITLE
Ryujinx loading slow

### ISSUE THREAD

**Issue by Marionut84** [Ecks1337/RyuSAK#25]
Ryujinx is taking forever to load games now that I added shaders from ryusak. Some games that worked perfectly before won't even load at all or take so long that it freezes or if it does load after a long period of time its very choppy.

**Comment by offbrandd** [comment]
Sort of bump on this. Takes long to load shaders (expected) but when it's done I typically have to restart ryujinx a couple of times to get the game open. May be a ryujinx issue however

**Comment by Marionut84** [comment]
Have you found that it Works for you after restarting a couple of times? Mine doesn't seem to be working much at all On most or all games now. I understand that some people have issues with the shaders taking a long time loading but I never had that problem before. Mine always seemed to load very fast no matter what. Only when I 1st originally installed it did it take a bunch of time But ever since like the 1st week it's worked perfectly for the most part With some tweaking here and there and updates of course But it always seemed to load the shaders very very quickly almost instant every other time. Just until recently when I installed ryusak did it start taking forever And I really didn't get that many shaders from them. Almost every game, I had more shaders than the program did and a shader list had already been submitted for most games already so I couldn't upload my additionals. I know I went through and did everything correctly. So not sure why now everything is not loading Or gets stuck and freezes. It worked perfectly a few days ago. Maybe it's the new update of ryujinx. I'm really lost at this point. I don't know a ton about this stuff but I've picked up a what I think is quite a bit over the last year and I'm stumped. I know it's not me doing anything wrong. At least I don't think. So Not sure Where to go from here? Any feedback from anyone would be greatly appreciated. I've already tried uninstalling and reinstalling everything but I think the damage has already been done. Unless I do a complete hard reset on all the programs(which I dont want to do) or theres a fix, my ryujinx is basically dead in the water now over one little, what seemed liked harmless program. Not blaming ryusak at all either. Just not sure what else the problem could be since everything's been working up until I installed it. Very frustrated. Was just remarking to my kid the other day that ryuJinx was amazing and I almost love it more and its almost better than yuzu. On a side note up until a few days ago both my emulators worked great and now I'm having major issues on both. After installing ryusak for ryuJinx and Yuzu Mod Downloader for Yuzu. Again not blaming the programs it's just what are the odds that everything's been working great for months And now both emulators are down for me since installing 2 companion programs.
